src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/DateFunctionsIT.java

```diff
@@ -201,19 +207,18 @@ public void secondOfMinute() {
         }
     }
 
-    private SearchHit[] query(String select, String... statements) {
+    private SearchHit[] query(String select, String... statements) throws IOException {
         return execute(select + " " + FROM + " " + String.join(" ", statements));
     }
 
-    private SearchHit[] execute(String sql) {
-        try {
-            SearchDao searchDao = MainTestSuite.getSearchDao();
-            SqlElasticSearchRequestBuilder select = (SqlElasticSearchRequestBuilder) searchDao.explain(sql).explain();
-            return ((SearchResponse) select.get()).getHits().getHits();
-        } catch (SQLFeatureNotSupportedException | SqlParseException e) {
-            throw new ParserException("Unable to parse query: " + sql);
-        }
+    private SearchHit[] execute(String sqlRequest) throws IOException {
```

Comment:
Not a blocker: I think this code is now re-used in multiple classes, would be good to move to the base class.

Reply:
Sure, will take this in the next commit. Looks like it needs a lot of refactoring. Will mark it with `TODO`.